src/App.js

```diff
@@ -10,31 +11,74 @@ function Profile(props) {
     <div className="Profile">
       <Avatar pic={props.data.profileUrl} />
       <div className="UserInfo">
-        {props.data.firstName} {props.data.lastName}
-        <div>Email: {props.data.email}</div>
-        <div>Age: {props.data.age}</div>
+        <div className="UserName">{props.data.userName}</div>
+        <p />
+        <ul>
+          <li>
+            Name: {props.data.firstName} {props.data.lastName}
+          </li>
+          <li>Email: {props.data.email}</li>
+          <li>Age: {props.data.age}</li>
+        </ul>
       </div>
     </div>
   );
 }
 
 function FavoriteRepos(props) {
   const items = props.data;
-  const listFavorites = items.map((favorites) => (
-    <li key={favorites.toString()}>
-      {favorites}
+  const listFavorites = items.map((favorites, index) => (
+    <li key={index}>
+      {favorites.name}
+      <br />
+      <div className="Description">
+        {favorites.description}
+        <br />
+        Update on <ConvertToDate time={favorites.updateTimestamp} />
+      </div>
       <hr />
     </li>
   ));
   return <ul>{listFavorites}</ul>;
 }
 
+function MyProjects(props) {
+  const items = props.data;
+  const listProjects = items.map((projects, index) => (
+    <li key={index}>
+      {projects.name}
+      <br />
+      <div className="Description">
+        {projects.description}
+        <br />
+        Update on <ConvertToDate time={projects.updateTimestamp} />
+      </div>
+      <hr />
+    </li>
+  ));
+  return <ul>{listProjects}</ul>;
+}
+
```

Comment:
please check this two component `FavoriteRepos` and `MyProjects`, don't you think they look familiar?

Reply:
may be we could use `RepoList` and take props `data`
```javascript
<RepoList data={favoriteList} />
<ReposList data={projects />
```
so we could re-use `ReposList`

Reply:
or 
```javascript
const FavList = <RepoList data={favoriteList} />;
const MyProject = <RepoList data={projects} />;
```
the purpose is used to DRY, maybe in the future you think they are not the same data structure, you could split easily

Reply:
thanks, I will refactor it.
use common component when display content and remove duplicate component.